webapp/channels/src/components/copy_text.tsx

```diff
@@ -2,14 +2,12 @@
 // See LICENSE.txt for license information.
 
 import React, {useCallback} from 'react';
-import {FormattedMessage} from 'react-intl';
+import {useIntl} from 'react-intl';
 
-import OverlayTrigger from 'components/overlay_trigger';
-import Tooltip from 'components/tooltip';
-
-import Constants from 'utils/constants';
 import {copyToClipboard} from 'utils/utils';
 
+import WithTooltip from './with_tooltip';
```

Comment:
instead of relative importing can you use absolute like `import X from 'components/x'